linkerd/failure-accrual/src/main/scala/io/buoyant/linkerd/failureAccrual/SuccessRateWindowedInitializer.scala

```diff
@@ -0,0 +1,23 @@
+package io.buoyant.linkerd.failureAccrual
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.conversions.time._
+import com.twitter.finagle.service.exp.FailureAccrualPolicy
+import io.buoyant.linkerd.{BackoffConfig, FailureAccrualConfig, FailureAccrualInitializer}
+
+class SuccessRateWindowedInitializer extends FailureAccrualInitializer {
+  val configClass = classOf[SuccessRateWindowedConfig]
+  override def configId = "io.l5d.successRateWindowed"
+}
+
+object SuccessRateWindowedInitializer extends SuccessRateWindowedInitializer
+
+case class SuccessRateWindowedConfig(
+  sr: Double,
```

Comment:
is `sr` too cryptic for a config param?  do you think `successRate` would be too verbose?